<launch>
    <node name="location_corrector_node" pkg="location_corrector" type="location_corrector_node" output="screen">
        <param name="map_frame" type="string" value="map"/>
        <param name="unadjust_map_frame" type="string" value="map_carto"/>
        <param name="child_frame" type="string" value="base_link"/>
        <!-- 在 unadjust_map_frame 中的坐标，对称中心点 -->
        <rosparam command="load" file="$(env SLAMROBOT_HOME)/data/battleground.pbstream.yaml"/>
    </node>
</launch>
